app/views/console/settings/index.phtml

```diff
@@ -473,7 +473,33 @@ $smtpEnabled = $this->getParam('smtpEnabled', false);
                                     <input name="teamId" data-ls-attrs="id=leave-teamId-{{member.$id}}" type="hidden" data-ls-bind="{{console-project.teamId}}">
                                     <input name="membershipId" data-ls-attrs="id=leave-membershipId-{{member.$id}}" type="hidden" data-ls-bind="{{member.$id}}">
 
-                                    <button class="danger">Leave</button>
+                                    <button data-ls-if="1 === {{members.memberships.length}}" class="danger" disabled>Leave</button>
+                                    <button data-ls-if="1 < {{members.memberships.length}}" class="danger">Leave</button>
```

Comment:
This is for non-me users. Should the text be `Kick` or `Remove`?

Reply:
Good call. I changed both to `Remove` for consistency